packages/editor-ui/src/views/canvasHelpers.ts

```diff
@@ -728,7 +729,7 @@ export const getFixedNodesList = (workflowNodes: INode[]) => {
 	});
 
 	if (!hasStartNode) {
-		nodes.push({...DEFAULT_START_NODE});
+		nodes.push({...DEFAULT_START_NODE, id: uuidv4() });
```

Comment:
ref previous comment
```suggestion
		nodes.push({...DEFAULT_START_NODE, id: uuid() });
```